charts/validation/values.yaml

```diff
@@ -0,0 +1,14 @@
+# Default values for validation.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: ghcr.io/cdcgov/phdi/validation
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
```

Comment:
I believe tag should have some sort of value so we don't automatically get latest for the repo.

Reply:
Per the comment there I think it defaults to the version in chart.yaml, so it wouldn't be latest it'd be 1.0.8 so I think this is ok?

Reply:
+1 to Gordon, this should default to 1.0.8 as written.

Though I wonder if it's worth defaulting to `latest`? Otherwise we could easily get stuck behind versions since I don't imagine we'll update the Helm charts terribly often.